crates/store/re_video/src/decode/mod.rs

```diff
@@ -1,4 +1,82 @@
 //! Video frame decoding.
+//! =========================
+//!
+//! Whirlwind tour of how to interpret picture data (from a Video perspective)
+//! ---------------------------------------------------------------------------------
+//!
+//! Extracted from the [av1 codec wiki](https://wiki.x266.mov/docs/colorimetry/intro) and other sources.
+//! Follows the trail of information we get from our AV1 decoder.
+//!
+//! ### How to get from YUV to RGB?
+//!
+//! Things to know about the incoming yuv data:
+//! * `picture.bit_depth()`
+//!   * is either 8 or 16
+//!   * that's how the decoder stores for us but the per component we have either 8 or 10 or 12 bits -> see `picture.bits_per_component()`
+//! * `picture.pixel_layout()`
+//!   * `4:0:0` greyscale
+//!   * `4:2:0` half horizontal and half vertical resolution for chroma
+//!   * `4:2:2` half horizontal resolution for chroma
+//!   * `4:4:4` full resolution for chroma
+//!   * note that the AV1 decoder gives us always (!) planar data
+//! * `picture.color_range()`
+//!   * yuv data range may be either `limited` or `full`
+//!   * `full` is what you'd naively expect, just full use up the entire 8/10/12 bits!
+//!   * `limited` means that only a certain range of values is valid
+//!      * weirdly enough, DO NOT CLAMP! a lot of software may say it's limited but then use the so-called foot and head space anyways to go outside the regular colors
+//!          * reportedly (read this on some forums ;-)) some players _do_ clamp, so let's not get too concerned about this
+//!      * it's a remnant of the analog age, but it's still very common!
+//! * TODO(andreas): It may actually be non-downsamples RGB as well, so skip the entire YUV conversion step! Haven't figured out yet how to determine that.
```

Comment:
```suggestion
//! * TODO(andreas): It may actually be non-downsampled RGB as well, so skip the entire YUV conversion step! Haven't figured out yet how to determine that.
```